test/scripts/jenkins_ci_group.sh

```diff
@@ -27,5 +27,23 @@ else
 
   export NODE_OPTIONS=--max_old_space_size=8192
 
+  echo " -> making hard link clones"
+  cd ..
+  cp -RlP kibana "kibana${CI_GROUP}"
+  rm "kibana${CI_GROUP}"/config/kibana.yml
+  cp kibana/config/kibana.yml "kibana${CI_GROUP}"/config/kibana.yml
```

Comment:
I'm doing this rm and cp in the integration-test repo because I'm actually modifying the kibana.yml file.  But I don't think our jobs modify it?  I think we just launch Kibana with cli params.  But it doesn't hurt.   Might be good to leave in case some future change actually does modify the kibana.yml.

Reply:
I push commit that removes it